### DISCUSSION TITLE
Fooocus Pro (???)

### DISCUSSION THREAD

**Discussion by feathered-arch** [lllyasviel/Fooocus#2113] [General]
https://fooocus.pro/contact-us/ <---- is this you??

If not... probably time to sue a bitch!

**Comment by mashb1t** [comment]
nope, see https://github.com/lllyasviel/Fooocus/issues/1908#issuecomment-1890396233
Also funny story: they use Pixabay as "backend", see my comment in the issue.

**Comment by feathered-arch** [comment]
lmao...
